server/src/test/java/org/candlepin/resteasy/JsonProviderTest.java

```diff
@@ -43,6 +49,13 @@
     @Mock private Configuration config;
     @Mock private ProductCurator productCurator;
 
+    private AtomicBoolean failed;
+
+    @Before
+    public void setUp() {
+        failed = new AtomicBoolean(false);
```

Comment:
You should be able to define this within the scope of the test using it; potentially declaring it as final for Java8 and earlier to reference it within the inner class.

Reply:
Done.